### PR TITLE
Feature: Add, Remove and Edit items

### PR DESCRIPTION
**Pull request by orozCoding**
In this branch I created the Add, Remove and Edit functions for the To Do List 👍 

- Add any item typed in the input filed ✔️ 
- Remove an item when clicking in the trash can icon ✔️ 
- Edit any item text/info if the text is clicked ✔️ 
- every time a task is added or removed, the array is updated, the indexes are sorted, and the data is stored in the localStorage ✔️

### CODE REVIEW

File: src/functions.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    description: 'You can remove any item with the button =>',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Enjoy!',
+    completed: false,
+    index: 4,
+  }];
+  return tasks;
```
Comment (TedLivist) [inline src/functions.js]: - [ ] As stated in the requirements kindly remove the hardcoded items in the tasks array.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @orozCoding,

Good job so far!
✔️ Nice documentation
✔️ Clean code
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] In your README, kindly add instructions on how users can set up your project locally. Also, remember to add `npm install` and `npm start` (or `npm run build`) as necessary commands.

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/functions.js (hunk 1 above)
Comment (orozCoding) [reply src/functions.js]: @TedLivist Thank you! Will update and push again ❤️

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🎉

### Hi @orozCoding 👋🏻, 

You did a tremendous job implementing the requested changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by orozCoding** [comment]
Thank you @Cathella

**Pull request merged**